### PR TITLE
Further leptonization

### PR DESCRIPTION
**Pull request by graahnul-grom**
About dialog: old - new:

about_dlg_old

about_dlg_new

License dialog:

license_dlg

Log window messages: old - new:

log_wnd_old

log_wnd_new

### CODE REVIEW

**Comment by graahnul-grom** [comment]
Could anybody tell me please,
1) Why the git version string in the About dialog
needs to be prefixed with "g", e.g. "ge1a91d3",
for short SHA1 "e1a91d3"?
2) Is it mandatory to display disclaimer like
"ABSOLUTELY NO WARRANTY" etc. in the log window?

File: schematic/src/gschem_about_dialog.c [hunk 1]
```diff
@@ -65,23 +65,48 @@ void about_dialog (GschemToplevel *w_current)
...
   }
 
-  gtk_show_about_dialog (
-      GTK_WINDOW (w_current->main_window),
-      "version",        version_string,
-      "logo",           logo,
-      "title",          _("About gschem"),
-      "comments",       _("gEDA: GPL Electronic Design Automation"),
-      "copyright",
-      /* TRANSLATORS: "ChangeLog" is a literal filename; please don't translate it. */
```
Comment (vzh) [inline schematic/src/gschem_about_dialog.c]: Please don't remove this line for translators!

File: schematic/src/gschem_about_dialog.c [hunk 2]
```diff
@@ -65,23 +65,48 @@ void about_dialog (GschemToplevel *w_current)
...
+
+  GtkWidget* dlg = gtk_about_dialog_new();
+  GtkAboutDialog* adlg = GTK_ABOUT_DIALOG (dlg);
+
+  gtk_about_dialog_set_version (adlg, version_string);
+  gtk_about_dialog_set_logo (adlg, logo);
+  gtk_about_dialog_set_comments (adlg, _("Lepton Electronic Design Automation"));
+
+  gtk_about_dialog_set_copyright (adlg,
+    _("Copyright © 1998-2018 by Ales Hvezda and the respective original authors.\n"
```
Comment (vzh) [inline schematic/src/gschem_about_dialog.c]: This line should be in line with *README*: 1998-201**7**

**Comment by vzh** [comment]
I am OK with suppressing the disclaimer in log.

**Comment by vzh** [comment]
I believe the prefix "g" was added by @peter-b in commit 49fb651 to underline this is the version number from `git`. I don't mind if you remove  it.

File: schematic/src/gschem_about_dialog.c (hunk 1 above)
Comment (graahnul-grom) [reply schematic/src/gschem_about_dialog.c]: Sorry about that! I'll resurrect it.

**Comment by graahnul-grom** [comment]
> I believe the prefix "g" was added by @peter-b in commit 49fb651 to underline this is
> the version number from git. I don't mind if you remove it.

What about replacing it with a `git: ` prefix string?

File: schematic/src/gschem_about_dialog.c (hunk 2 above)
Comment (vzh) [reply schematic/src/gschem_about_dialog.c]: IMO, the first line is about gEDA/gaf authors, and the second line about Lepton devs. So no, don't change the latter. Better fix README :-)

**Comment by vzh** [comment]
@graahnul-grom I'm fine with `git:`

**Pull request merged**